replicator/blip_messages.go

```diff
@@ -0,0 +1,41 @@
+package replicator
+
+import (
+	"fmt"
+
+	"github.com/couchbase/go-blip"
+)
+
+type BLIPMessageSender interface {
+	Send(s *blip.Sender) error
+}
+
+// SubChangesRequest is a strongly typed 'subChanges' request.
+type SubChangesRequest struct {
+	Continuous bool   // Continuous can be set to true if the requester wants change notifications to be sent indefinitely (optional)
+	Batch      uint16 // Batch controls the maximum number of changes to send in a single change message (optional)
+	Since      uint64 // Since represents the latest sequence ID already known to the requester (optional) // TODO: Need a more complex sequence number type
+	Filter     string // Filter is the name of a filter function known to the recipient (optional)
```

Comment:
Ah yeah, the channels one was quite buried in the replication protocol docs. Thanks!

Even worse, the `activeOnly` property isn't there! Opening a PR to update that doc now.

I'll add the `activeOnly` flag but default it to false, because it's specified in the PRD as a P1 for some use-cases, but it seems problematic in scenarios where clients can move between SG clusters.